seatunnel-connectors/seatunnel-connector-flink-druid/src/main/java/org/apache/seatunnel/flink/sink/DruidOutputFormat.java

```diff
@@ -0,0 +1,208 @@
+        String taskJSON = mapper.writeValueAsString(indexTask);
+        JSONObject jsonObject = JSON.parseObject(taskJSON);
+        jsonObject.remove("id");
+        jsonObject.remove("groupId");
+        jsonObject.remove("resource");
+        JSONObject spec = jsonObject.getJSONObject("spec");
+        spec.remove("tuningConfig");
+        jsonObject.put("spec", spec);
+        taskJSON = jsonObject.toJSONString();
+
+        URL url = new URL(this.coordinatorURL + "druid/indexer/v1/task");
+        HttpURLConnection con = (HttpURLConnection) url.openConnection();
+        con.setRequestMethod("POST");
+        con.setRequestProperty("Content-Type", "application/json");
+        con.setRequestProperty("Accept", "application/json, text/plain, */*");
+        con.setDoOutput(true);
+        try (OutputStream os = con.getOutputStream()) {
+            byte[] input = taskJSON.getBytes(StandardCharsets.UTF_8);
+            os.write(input, 0, input.length);
+        }
+        try (BufferedReader br = new BufferedReader(new InputStreamReader(con.getInputStream(), StandardCharsets.UTF_8))) {
+            StringBuilder response = new StringBuilder();
+            String responseLine;
+            while ((responseLine = br.readLine()) != null) {
+                response.append(responseLine.trim());
+            }
+            LOGGER.info("Druid write task has been sent, and the response is {}", response.toString());
+        }
+    }
+
+    private ParallelIndexSupervisorTask parallelIndexSupervisorTask(ParallelIndexIOConfig ioConfig, ParallelIndexTuningConfig tuningConfig) {
+        return new ParallelIndexSupervisorTask(
+                null,
+                null,
+                null,
+                new ParallelIndexIngestionSpec(
+                        new DataSchema(
+                                this.datasource,
+                                new TimestampSpec("ts", "auto", null),
+                                new DimensionsSpec(Collections.emptyList()),
+                                null,
+                                new UniformGranularitySpec(Granularities.HOUR, Granularities.MINUTE, false, null),
+                                null
+                        ),
+                        ioConfig,
+                        tuningConfig
+                ),
+                null
+        );
+    }
+
+    private ParallelIndexIOConfig parallelIndexIOConfig() {
+        return new ParallelIndexIOConfig(
+                null,
+                new InlineInputSource(this.data.toString()),
+                new CsvInputFormat(
+                        Arrays.asList("name", "ts"),
+                        "|",
+                        null,
+                        false,
+                        0
+                ),
+                false,
+                null
+        );
+    }
+
+    private ParallelIndexTuningConfig tuningConfig() {
+        return new ParallelIndexTuningConfig(
+                null,
```

Comment:
hi @asdf2014, does `ParallelIndexTuningConfig` has other `constructor`?

Reply:
Hi @leo65535 , thanks for your comment. Unfortunately, this is the only one. May I ask what's your concern